src/styles/blocks/catalog.scss

```diff
@@ -12,11 +14,18 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $transition-time, color $transition-time;
+
+    &:hover {
+      transform: scale(1.2);
+      color: $card-title-color;
```

Comment:
The color change for the card title should be applied specifically to the card title element, not to the whole card. Move the color change to the appropriate element.